ch04/calculatePi.go

```diff
@@ -26,21 +26,21 @@ func Pi(accuracy uint) *big.Float {
 		if k > int(accuracy) {
 			break
 		}
-		t1 := float64(float64(1) / float64(10*k+9))
```

Comment:
Precision gets lost due to these divisions, so changing to use .Quo instead

Reply:
Thanks @Jake-Convictional.

Reply:
Also put your code in the _2nd edition_ of the book: https://github.com/PacktPublishing/Mastering-Go-Second-Edition

Reply:
Sounds good, happy to be able to contribute!